pkg/splunk/enterprise/configuration.go

```diff
@@ -1607,3 +1644,71 @@ maxGlobalRawDataSizeMB = %d`, indexDefaults, defaults.MaxGlobalRawDataSizeMB)
 `, indexDefaults)
 	return indexDefaults
 }
+
+// validateProbe validates a generic probe values
+func validateProbe(probe *corev1.Probe) error {
```

Comment:
Is there an upper threshold for these values?

Reply:
We dont have any explicit upper limits.

Reply:
Should we set it just to safeguard ourselves?

Reply:
We can take this up in next sprint.